config/service_qos_config.go

```diff
@@ -464,6 +488,13 @@ var shannonServices = []ServiceQoSConfig{
 		sharedtypes.RPCType_COMET_BFT: {},
 	}),
 
+	// Namada TODO_TECHDEBT(@commoddity): Namada is not a conventional Cosmos SDK chain and likely requires a custom implementation.
```

Comment:
To my knowledge, Namada uses a fork of CometBFT and does not leverage the Cosmos SDK at all.

I'll probably see @adrianbrink in the bay area next week and will bring it up with him then.